### PR TITLE
jmx monitoring plugin

### PR DESCRIPTION
**Pull request by kovrus**
image

and 
image

image

### CODE REVIEW

**Comment by kovrus** [comment]
@joemoe pls check whether licensing is okey

File: CHANGES.txt [hunk 1]
```diff
@@ -5,6 +5,8 @@ Changes for Crate
 Unreleased
 ==========
 
+ - Enterprise Edition: Implemented the JMX monitoring plugin.
```
Comment (mfussenegger) [inline CHANGES.txt]: Not sure if this makes sense to our users

Comment (kovrus) [inline CHANGES.txt]: i am not sure either. it's already in docs, though that it can be somehow distinguished from other features

Comment (mfussenegger) [inline CHANGES.txt]: I think something along the lines of 

> Added functionality to monitor query runtime statistics via JMX. This feature can only be used with a enterprise license.

Would make more sense

Comment (joemoe) [inline CHANGES.txt]: I agree with @mfussenegger.

**Comment by kovrus** [comment]
@mfussenegger @joemoe fixup

**Pull request merged**